City-of-Helsinki/berth-reservations-admin
# VEN-1327 | Add custom pricing tables

## Description :sparkles:
- Add three pricing tables to the pricing page for custom berth prices 

## Issues :bug:

### Closes :no_good_woman:
[VEN-1327](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1327): Support custom prices

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️
- Update related tests

### Manual testing :construction_worker_man:
- Go to the pricing page 
- Under Venepaikka card, you should see three more tables for other custom prices 

## Screenshots :camera_flash:
![image](https://user-images.githubusercontent.com/23040926/119109377-c8c96780-ba29-11eb-9adf-132475c99b44.png)

## Additional notes :spiral_notepad:
